app/controllers/api/v1/game_play_apis_controller.rb

```diff
@@ -1,8 +1,18 @@
-# require 'pry'; binding.pry
+# require './././app/serializer/game_play_serializer'
```

Comment:
hey, if this line is not needed why not remove it entirely?

Reply:
Removed